turtlesim/include/turtlesim/turtle_frame.h

```diff
@@ -70,24 +71,20 @@ private slots:
   void clear();
   bool hasTurtle(const std::string& name);
 
-  bool clearCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&);
-  bool resetCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&);
-  bool spawnCallback(turtlesim::Spawn::Request&, turtlesim::Spawn::Response&);
-  bool killCallback(turtlesim::Kill::Request&, turtlesim::Kill::Response&);
```

Comment:
Is there a reason to replace these callback methods with lambdas? If not, please keep them as they were before tin order to not introduce unnecessary diffs.

Reply:
Because lambda is recommended style in ROS2

Reply:
> Because lambda is recommended style in ROS2

Can you please post a link where you have read that.

Anyway in this case keeping the diff minimal is more important. So please keep the callback methods as they were before.

Reply:
> Can you please post a link where you have read that.

I didn't read a post. I thought demos repo in ros2 is developed by recommended style. 
Because server in demos represents lambdas (https://github.com/ros2/demos/blob/df5ec3a477c683bbaf9b3a2ec064680a0d36829d/demo_nodes_cpp/src/services/add_two_ints_server.cpp#L42)

But to keep the original code I've modified the code.